python/main.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import random
+
+import hazelcast
+
+logging.basicConfig(level=logging.INFO)
+
+client = hazelcast.HazelcastClient(
+    cluster_members=["<EXTERNAL-IP>"],
+    use_public_ip=True,
```

Comment:
Why is this parameter needed? Won't it work if you don't set it to `True`?

Reply:
In Go, C++, and Python clients, we have decided to make this a first-class configuration element rather than a property. So, we thought it would be okay to not treat this as a tri-state boolean but rather as an on/off switch. 

So, if the user wants to enable the discovery using this logic, this flag needs to be set to `True` in those languages.

Reply:
In Java, you don't need to set this property, because it's automatically detected. Is that the case for Python/C++/Go?

Reply:
No, it is a simple on/off flag in Python/C++/Go. The detection logic is not implemented.

Reply:
Do you think that this auto-detection logic must be implemented in those languages as well? Maybe, we can do that based on your input. At least we can discuss it

Reply:
I see. I think the auto-detection logic is not critical at the moment. Let's keep it as it is.